src/main/java/org/elasticsearch/index/query/SimpleQueryStringBuilder.java

```diff
@@ -114,24 +125,46 @@ public SimpleQueryStringBuilder flags(SimpleQueryStringFlag... flags) {
         this.flags = value;
         return this;
     }
+    
+    SimpleQueryStringBuilder flags(int flags) {
```

Comment:
how would java api users set these flags? seems like an int here is not very user friendly, also why is this method package private?